docs/src/main/sphinx/connector/teradata.md

```diff
@@ -0,0 +1,214 @@
+
+```{include} jdbc-authentication.fragment
+```
+
+### Multiple Teradata databases
+
+You can have as many catalogs as you need, so if you have additional Teradata
+databases, simply add another properties file to etc/catalog with a different
+name, making sure it ends in .properties. 
+For example, if you name the property file sales.properties, Trino creates a 
+catalog named sales using the configured connector.
+
+## Type mapping
+
+Because Trino and Teradata each support types that the other does not, this
+connector {ref}`modifies some types <type-mapping-overview>` when reading data.
+Refer to the following sections for type mapping when reading data from
+Teradata to Trino.
+
+### Teradata type to Trino type mapping
+
+The connector maps Teradata types to the corresponding Trino types following
+this table:
+
+:::{list-table} Teradata type to Trino type mapping
+:widths: 40, 40, 20
+:header-rows: 1
+
+* - Teradata type
+  - Trino type
+  - Notes
+* - `TINYINT`
+  - `TINYINT`
+  -
+* - `SMALLINT`
+  - `SMALLINT`
+  -
+* - `INTEGER`
+  - `INTEGER`
+  -
+* - `BIGINT`
+  - `BIGINT`
+  -
+* - `REAL`
+  - `DOUBLE`
+  -
+* - `DOUBLE`
+  - `DOUBLE`
+  -
+* - `FLOAT`
+  - `DOUBLE`
+  -
+* - `NUMBER(p, s)`
+  - `DECIMAL(p, s)`
+  - 
+* - `NUMERIC(p, s)`
+  - `DECIMAL(p, s)`
+  - 
+* - `DECIMAL(p, s)`
+  - `DECIMAL(p, s)`
+  - 
+* - `CHAR(n)`
+  - `CHAR(n)`
+  -
+* - `CHARACTER(n)`
+  - `CHAR(n)`
+  -
+* - `VARCHAR(n)`
+  - `VARCHAR(n)`
+  - 
+* - `DATE`
+  - `DATE`
+  -
+:::
+
+No other types are supported.
+
+### Trino type to Teradata type mapping
+
+The connector maps Trino types to the corresponding Teradata types following
+this table:
+
+:::{list-table} Trino type to Teradata type mapping
+:widths: 40, 40, 20
+:header-rows: 1
+
+* - Trino type
+  - Teradata type
+  - Notes
+* - `TINYINT`
+  - `SMALLINT`
+  -
+* - `SMALLINT`
+  - `SMALLINT`
+  -
+* - `INTEGER`
+  - `INTEGER`
+  -
+* - `BIGINT`
+  - `BIGINT`
+  -
+* - `REAL`
+  - `FLOAT`
+  -
+* - `DOUBLE`
+  - `DOUBLE`
+  - 
+* - `DATE`
+  - `DATE`
+  -
+::::
+
+No other types are supported.
+
+```{include} jdbc-type-mapping.fragment
+```
+
+## Querying Teradata
+
+The Teradata connector provides a schema for every Teradata database. You can
+see the available Teradata databases by running SHOW SCHEMAS:
+
+```sql
+SHOW SCHEMAS FROM teradata;
+```
+
+If you have a Teradata database named sales, you can view the tables in this
+database by running SHOW TABLES:
+
+```sql
+SHOW TABLES FROM teradata.sales;
+```
+
+You can see a list of the columns in the orders table in the sales database
+using either of the following:
+
+```sql
+DESCRIBE teradata.sales.orders;
+SHOW COLUMNS FROM teradata.sales.orders;
+```
+
+Finally, you can access the orders table in the sales database:
+
+```sql
+SELECT * FROM teradata.sales.orders;
+```
+
+## SQL support
+
+The connector provides read access to data and metadata in
```

Comment:
This is inconsistent with intro and type mapping .. those sections imply that some DDL statements are suppported.. please correct and use wording/formmatting like in other connectors

Reply:
Updated with correct formatting and added about supported `DDL` statements.